.github/scripts/create_dbt_projects.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -e
+
+echo "Creating dbt projects..."
+
+init_dbt_project() {
+  local project_name=$1
+  local profile_selection=$2
+  local target_selection=$3
+  local host=$4
+  local username=$5
+  local password=$6
+
+  dbt init "$project_name" <<EOF
+$profile_selection
+$target_selection
+$host
+
+$username
+$password
+
+
+
+
+
+
+
+
```

Comment:
[NIT]
Remove empty lines

Reply:
Both of these are on purpose, it's simulating the user input required